### PR TITLE
Removed the docs about Debug

### PR DESCRIPTION
**Pull request by wouterj**
The doc does nothing more than showing a `composer install` command and linking to a class on github.

#SymfonyHackathon

### CODE REVIEW

File: components/class_loader/debug_class_loader.rst [hunk 1]
```diff
@@ -3,6 +3,7 @@ Debugging a Class Loader
 
 .. caution::
 
-    The ``DebugClassLoader`` from the ClassLoader component was deprecated
-    in Symfony 2.5 and removed in Symfony 3.0. Use the
-    :ref:`DebugClassLoader provided by the Debug component <component-debug-class-loader>`.
```
Comment (OskarStark) [inline components/class_loader/debug_class_loader.rst]: Do we need this old deprecation?

Comment (javiereguiluz) [reply components/class_loader/debug_class_loader.rst]: We may remove it ... but I think that keeping it (only in 3.4) is also OK.

**Review by javiereguiluz — APPROVED** [review]

**Comment by wouterj** [comment]
Redirection added, ready to merge imho

**Review by OskarStark — APPROVED** [review]

**Comment by OskarStark** [comment]
Thanks Wouter.

**Pull request merged**